tests/kafkatest/services/connect.py

```diff
@@ -131,7 +130,7 @@ def start_and_wait_to_load_plugins(self, node, worker_type, remote_connector_con
                                err_msg="Never saw message indicating Kafka Connect finished startup on node: " +
                                        "%s in condition mode: %s" % (str(node.account), self.startup_mode))
 
-    def start_and_wait_to_start_listening(self, node, worker_type, remote_connector_configs):
+    def start_and_wait_to_load_rest(self, node, worker_type, remote_connector_configs):
```

Comment:
we don't need to rename this method, as per @rhauch's earlier comment.

Reply:
Sure, reverting it back.